_posts/2019-06-28-newsletter.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Origami Newsletter, June 2019
+description: This issue announces a new team member, covers improvements to our operational documentation, and some projects that we built in our 10% time.
+author: Rowan Manning
+tags:
+  - Newsletter
+---
+
+**TL;DR:** This issue announces a new team member, covers improvements to our operational documentation, and some projects that we built in our 10% time.
+
+## Top three things
+
+These are some of the bigger things we've done over the last month.
+
+### New Team Member
+
+We're super pleased to announce that Chee Rabbits will be joining the Origami team starting in July! 🎉 Their extensive knowledge of the web, and their general enjoyment of helping people makes them a great fit for the team. We were impressed with all of the internal candidates that we interviewed, so I'd like to thank them again for their time and enthusiasm. Also a big thanks to Customer Products for being supportive of Chee's move.
+
+### Operational Documentation
+
+We've spent a lot of time this month improving our operational documentation, using the excellent [System Operability Score (SOS) app](https://sos.in.ft.com/). Our experience using SOS has been great, and it definitely encouraged us to stop putting off improving our service's operability. Also we're pretty proud to be at the top of the team leaderboard 😉
+
+### 10% Time
+
+We recently introduced 10% time to the team, and this has been the first month when we managed to kick off some of the projects and experiments that we otherwise end up side-lining. We have two projects on the go: The [Origami ScreenCap Service](https://origami-screencap.ft.com/v1) allows you to automate screenshots, and was built to experiment with demo thumbnails in our registry; [Origami Migration Guru (OMG)](https://github.com/Financial-Times/origami-migration-guru) is a command-line tool to help plan major version bumps on a project which has lots of dependents.
```

Comment:
What does side-lining mean?

Reply:
Like, putting to the side and never prioritising. I'll reword